components/script/dom/htmliframeelement.rs

```diff
@@ -491,16 +492,8 @@ impl HTMLIFrameElementMethods for HTMLIFrameElement {
     }
 
     // https://html.spec.whatwg.org/multipage/#attr-iframe-allowfullscreen
```

Comment:
This should link to https://html.spec.whatwg.org/multipage/embedded-content.html#dom-iframe-allowfullscreen .